sections/04_study.md

```diff
@@ -241,114 +241,61 @@ this was easy to simulate with a computational model. As we learn to better
 visualize and analyze the hidden nodes within deep learning models, our
 understanding of TF binding motifs and dynamics will likely improve.
 
-### Promoters, enhancers, and related epigenomic features
-
-Transcriptional control is undoubtedly a vital, early part of the regulation of
-gene expression. An abundance of sequence and associated functional data (e.g.
-ENCODE [@tag:Consortium2012_encode] and ExAC [@doi:10.1038/nature19057]) exists
-across species. At the same time, studies of gene regulation have often focused
-on the protein (binding) rather than the promoter level
-[@doi:10.1093/bib/4.1.22], perhaps due to the ill-defined nature of
-_cis_-regulatory elements (CREs). A promoter itself can be seen as an assemblage
-of "active" binding sites for transcription factors interspersed by
-less-characterized and perhaps functionally silent spacer regions. However, the
-sequence signals that control the start and stop of transcription and
-translation are still not well understood, compounded by incomplete
-understanding of alternative transcripts and the context for these alternatives.
-Sequence similarity is poor even between functionally correlated genes. While
-homologs might be studied for insight, they may not exist or may be just as
-poorly characterized.
-
-Recognizing enhancers presents additional challenges. Enhancers may be up to one
-million base pairs upstream or downstream from the affected promoter on either
-strand and even within the introns of other genes [@doi:10.1038/nrg3458]. They
-do not necessarily operate on the nearest gene and may affect multiple genes.
-Their activity is frequently tissue- or context-specific. Many
-enhancers display modest or no conservation across species. There is no
-universal enhancer sequence signal or marker for enhancers, and some literature
-suggests that enhancers and promoters may be just categories along a spectrum
-[@doi:10.1016/j.tig.2015.05.007]. One study [@doi:10.1101/gr.173518.114] even
-showed that only 33% of predicted regulatory regions could be validated, while a
-class of "weak" predicted enhancers were strong drivers of expression. Yet there
-is growing evidence for their vast ubiquity, making them possibly the
-predominant functional non-coding element. Thus, identifying enhancers is
-critical yet the search space is large.
-
-While previous (non-deep learning) approaches have steadily improved
-promoter prediction, there is little consensus on the best approach and
-performance is poor. Typically algorithms will recognize only half of all
-promoters, with an accompanying high false positive rate
-[@doi:10.1101/gr.7.9.861]. Methods with better sensitivity generally do so at
```

Comment:
There have been too many advances in available data and methodology, both computational and biochemical, in the last 20 years for a review that old to be useful, especially in terms of describing what is "typical".